### ISSUE TITLE
Error: terminfo for xterm-256color has invalid magic number

### ISSUE THREAD

**Issue by TheArcaneBrony** [Qix-/terminfo#5]
```
/home/User/git/NewProject12/node_modules/terminfo/index.js:40
                throw new Error('terminfo for ' + name + ' has invalid magic number');
        ^
Error: terminfo for xterm-256color has invalid magic number
    at parseTermBuffer (/home/User/git/NewProject12/node_modules/terminfo/index.js:40:9)
    at module.exports (/...)
    at Function.printLogo (...)
```

Environment: NixOS 24.11

**Comment by Qix-** [comment]
Yeah :( Looks like terminfo database has changed their formats at some point over the last couple of years.

See https://github.com/Qix-/better-exceptions/issues/117.

I unfortunately don't have a lot of time to devote to this; if someone wants to investigate and report back (or better, PR) I'd be happy to look into a fix or cut a release.

**Comment by TheArcaneBrony** [comment]
as far as i found, the only difference is the size of integer fields being 32 bits instaed of 16 when the magic byte is 0x21e instead of 0x11a

Some code i modified to work with this is available at https://github.com/spacebarchat/server/blob/dev/automod/src/util/util/os-utils/TermInfo/parseTerminfo.ts

**Comment by Qix-** [comment]
Oh neat, thanks for the information! PR welcome if anyone is up for it!
